CMakeLists.txt

```diff
@@ -122,8 +123,8 @@ include_directories(SYSTEM ${LIBRT_INCLUDE_DIR})
 
 include_directories(SYSTEM /usr/include)
 include_directories(SYSTEM /usr/local/include)
-link_directories(SYSTEM /usr/lib) #Access g2o
-link_directories(SYSTEM /usr/local/lib) #Access g2o
+# link_directories(SYSTEM /usr/lib) #Access g2o
```

Comment:
What is this g2o?

Reply:
They can be safely removed if you want.

Reply:
Would prefer to remove them then. I can do that after the merge though. There is still much to be done in cleanup effort anyway.

Reply:
Yeah, you can do it after the merge.